deployments/hadoop-yarn/ansible/config/zeppelin.yml

```
@@ -21,15 +21,15 @@
 #
 
 # Zeppelin vars
-zepvers: "0.10.0"
-zepname: "zeppelin-0.10.0-bin-all"
+zepvers: "0.10.1-gaia-dmp-0.1"
+zepname: "zeppelin-0.10.1-gaia-dmp-0.1"
+zepurl: "https://object.arcus.openstack.hpc.cam.ac.uk/swift/v1/AUTH_e216e6b502134b6185380be6ccd0bf09/archive/zeppelin-0.10.1-gaia-dmp-0.1.tar.gz"
```

Does this work if we use references ?
```suggestion
zepvers: "0.10.1-gaia-dmp-0.1"
zepname: "zeppelin-{{zepvers}}"
zepurl: "https://object.arcus.openstack.hpc.cam.ac.uk/swift/v1/AUTH_e216e6b502134b6185380be6ccd0bf09/archive/{{zepname}}.tar.gz"
```

---

I remember having problems getting a cascade of variables referencing other variables to work, but it would be much cleaner and less error prone if it did work.